### PR TITLE
re-add ContentType to combiner payload

### PR DESCRIPTION
**Pull request by RafalSkolasinski**
Executor: fix missing ContentType in combiner payload

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **rafalskolasinski**
You can assign the PR to them by writing `/assign @rafalskolasinski` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by seldondev** [comment]
Fri Aug 28 14:04:51 UTC 2020
The logs for [pr-build] [1] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2341/1.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2341 --build=1

**Comment by seldondev** [comment]
Fri Aug 28 14:05:04 UTC 2020
The logs for [lint] [2] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2341/2.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2341 --build=2

**Comment by seldondev** [comment]
Fri Aug 28 14:05:05 UTC 2020
The logs for [integration] [3] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2341/3.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2341 --build=3

**Comment by seldondev** [comment]
Fri Aug 28 14:13:46 UTC 2020
The logs for [lint] [5] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2341/5.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2341 --build=5

**Comment by seldondev** [comment]
Fri Aug 28 14:13:44 UTC 2020
The logs for [pr-build] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2341/4.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2341 --build=4

**Comment by seldondev** [comment]
Fri Aug 28 14:14:37 UTC 2020
The logs for [integration] [6] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2341/6.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2341 --build=6

**Comment by seldondev** [comment]
@RafalSkolasinski: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | b0c93d40a79c61281e8602584c5fd491196c8419 | [link]() | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by RafalSkolasinski** [comment]
Looks like the old good flakiness failure
/retest

**Comment by seldondev** [comment]
Fri Aug 28 16:45:34 UTC 2020
The logs for [integration] [7] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2341/7.log

impatient try  
jx get build logs SeldonIO/seldon-core/PR-2341 --build=7

File: executor/api/rest/utils.go
Comment (axsaucedo) [inline executor/api/rest/utils.go:71]: Nice one! This makes sense - we shoudl document somewhere that there is assumption that we will always return the content type of the first element (just to make sure it's explicit for users in case they don't know what the content type rules are. I assume this is basically how the java engine used to do it? If so we can merge this as it's the same as before and make sure we extend with the docs addition

**Review by axsaucedo — CHANGES_REQUESTED** [review]
Looks good, added a question/comment, should be good to merge once confirmed!

File: executor/api/rest/utils.go
Comment (RafalSkolasinski) [reply executor/api/rest/utils.go:71]: I believe that content type of what comes in every message to combiner is anyway determined by content type of initial message send by user.

After all it's the initial message independently sent to every child and output from those goes to the combiner.

This is of course under assumption thst content type cannot change in the Python Wrapper but I think we do not give option for that...

**Pull request merged**
